src/server/gc_worker/gc_manager.rs

```diff
@@ -527,12 +526,12 @@ impl<S: GcSafePointProvider, R: RegionInfoProvider> GcManager<S, R> {
         // TODO: Find a better way to handle errors. Maybe we should retry.
         debug!(
             "trying gc"; "region_id" => ctx.get_region_id(), "region_epoch" => ?ctx.region_epoch.as_ref(),
-            "end_key" => next_key.as_ref().map(DisplayValue)
+            "end_key" => ?next_key
```

Comment:
Does `next_key` contain user data?

Reply:
`next_key` is a `txn_types::Key`, and the filtering is done in the type's `Debug` impl. I agree though the inconsistency with normal `Vec<u8>` is confusing and probably error prone..